pkg/instrumentation/jaeger.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instrumentation
+
+import (
+	"contrib.go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/trace"
+)
+
+// createJaegerExporter creates a trace data exporter for Jaeger.
+func (s *Service) createJaegerExporter() error {
+	var err error
+
+	log.Debug("creating Jaeger exporter...")
+
+	cfg := jaeger.Options{
+		ServiceName:       ServiceName,
+		CollectorEndpoint: opt.Collector,
+		AgentEndpoint:     opt.Agent,
+
+		Process: jaeger.Process{ServiceName: ServiceName},
+		OnError: func(err error) { log.Error("%v", err) },
+	}
+
+	if s.jexport, err = jaeger.NewExporter(cfg); err != nil {
+		return instrumentationError("failed to create Jaeger exporter: %v", err)
+	}
+
+	return nil
+}
+
+// startJaegerExporter 'starts' the Jeager exporter by registering it.
+func (s *Service) startJaegerExporter() {
+	if s.jexport != nil {
+		if opt.Trace != Disabled {
```

Comment:
nit: you could drop one level of indentation by using `&&` (?)

Reply:
Fixed.